data-model/fdo-type/annotation/0.3.0/schema/annotation.json

```diff
@@ -390,6 +389,56 @@
     "ods:batchID": {
       "type": "string",
       "description": "Internally generated PID to identify the batch the annotation was generated by"
+    },
+    "ods:mergingStatus": {
```

Comment:
Why not capture if a decision is pending? If a machine wants to find if a decision has been made, it would have to do a null check. 
something like `ods:DecisionPendingForMerging`
`

Reply:
hmm yes you might be right, so when we create an annotation with motivation adding/editing we will fill this status with DecisionPending. We can then easily filter all annotations which require a decision

Reply:
Hmm it should be there, just checked and I can see.
```
      "enum": [
        "ods:DecisionPendingForMerging",
        "ods:RejectedForMerging",
        "ods:ApprovedForMerging"
      ],
```

Reply:
yep I see it, looks good to me!

Reply:
That would work, It looks like as end status for the annotation we really are capturing the mergin decision status, We could then also rename ods:mergingStatus to ods:mergingDecisionStatus with as enum: ods:pending, ods:rejected, ods:approved.

I wonder if we need an extra state for the object ods:status:merged? That may make it also easier for the front-end to decide whether to show an annotation. By default we may not want to show merged annotations with the digital specimen anymore. Or should they be tombstoned after merging? I think they are not active anymore as they cannot be further decided on for merging and should not be commented on further.  I guess annotations that were rejected for merging,  would also be tombstoned.

Reply:
I agree, let's simplify, and call it `ods:mergingDecisionStatus`. I will push the changes in a moment.

I would say they get tombstoned, the annotation is no longer necessary as it has now become part of the object. After merging we can tombstone the annotation based on the `ods:mergingDecisionStatus` somebody can check if it was approved or rejected.

Reply:
ok if they get tombstoned then the example needs to be changed as well.